### PR TITLE
Support text formatting in |error|

### PR DESCRIPTION
**Pull request by mia-pi-git**

### CODE REVIEW

File: js/client-chat.js [hunk 1]
```diff
@@ -1404,7 +1404,8 @@
 					break;
 
 				case 'error':
-					this.$chat.append('<div class="notice message-error">' + BattleLog.escapeHTML(row.slice(1).join('|')) + '</div>');
+					// eslint-disable-next-line no-undef
+					this.$chat.append('<div class="notice message-error">' + formatText(row.slice(1).join('|'), true) + '</div>');
```
Comment (Zarel) [inline js/client-chat.js:1408]: is `formatText` not defined here?

Comment (mia-pi-git) [reply js/client-chat.js:1408]: It works locally. Not sure what's with ESLint.

Comment (Zarel) [reply js/client-chat.js:1408]: isn't ``formatText`` already used elsewhere in client-chat?

Comment (mia-pi-git) [reply js/client-chat.js:1408]: Not on a cursory search. The only usage i see is in `battle-log.ts`.

Comment (Zarel) [reply js/client-chat.js:1408]: aha, looks like you should be using `BattleLog.parseMessage`. You should be adding a `trusted` param _there_.

Comment (mia-pi-git) [reply js/client-chat.js]: Done.

**Comment by Zarel** [comment]
Thanks!

**Pull request merged**